script/shared_functions

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+#
+# Shared functions
+
+# shellcheck source=script/ask
+source ./ask
+
+# color variables
+red=$(tput setaf 1)
```

Comment:
I ❤️ the addition of colors.  This is going to make the command line output more readable.